packages/react/testing-library/src/__tests__/rerender.test.jsx

```diff
@@ -45,7 +45,7 @@ test('rerender will flush pending hooks effects', async () => {
   const { rerender } = render(<Component />);
   const { findByText } = rerender(<Component />);
   vi.spyOn(lynx.getNativeApp(), 'callLepusMethod');
-  const callLepusMethod = lynxEnv.backgroundThread.lynx.getNativeApp().callLepusMethod;
+  const callLepusMethod = lynxTestingEnvironment.backgroundThread.lynx.getNativeApp().callLepusMethod;
```

Comment:
is callLepusMethod a public API? If so i'd love to rename it

Reply:
No, `callLepusMethod` is an internal API, it can be used to call a main-thread function on background thread, its implementation is here:

https://github.com/lynx-family/lynx/blob/6ac4d4e04153f0ed8e88e8ba7330d86dd1c21a9f/core/renderer/template_assembler.cc#L3014-L3026